src/cache/precomputed_quantities.jl

```diff
@@ -333,8 +330,11 @@ ts_sgs(thermo_params, moisture_model, specific, K, Φ, p) = thermo_state(
     p,
 )
 
-function eddy_diffusivity_coefficient(C_E, norm_v_a, z_a, p)
-    p_pbl = 85000
+function eddy_diffusivity_coefficient(D₀, H, z_sfc, z)
+   return D₀ * exp(-(z - z_sfc) / H)
+end
+function eddy_diffusivity_coefficient(C_E, norm_v_a, z_a, p, p_sfc)
```

Comment:
Let's keep this vertical diffusion unchanged so it's the same as the Held Suarez literature. We should not use it for any topography case.